src/model/model.cpp

```diff
@@ -2,11 +2,7 @@
 
 Point::Point(int x, int y) : x{x}, y{y} {}
 
-bool operator == (const Point & a, const Point & b) {
-    return a.x == b.x && a.y == b.y;
-}
-
-double operator - (const Point & a, const Point & b) {
-    return sqrt(pow(b.x - a.x, 2.0) + pow(b.y - a.y, 2.0));
-}
-
+bool operator == (const Point & a, const Point & b) {return a.x == b.x && a.y == b.y;}
+double operator / (const Point & a, const Point & b) {return sqrt(pow(b.x - a.x, 2.0) + pow(b.y - a.y, 2.0));}
```

Comment:
why divide points?

Reply:
I've abused the `/` operator for calculating distances, since it's never going be used elsewhere.

yes, I know that's bad design